Sources/ApolloCodegenLib/AST/SortedSelections.swift

```diff
@@ -0,0 +1,131 @@
+import Foundation
+import OrderedCollections
+
+struct SortedSelections: Equatable, CustomDebugStringConvertible {
+  typealias Selection = CompilationResult.Selection
+  typealias Field = CompilationResult.Field
+  typealias TypeCase = CompilationResult.SelectionSet
+  typealias Fragment = CompilationResult.FragmentDefinition
+
+  fileprivate(set) var fields: OrderedDictionary<String, Field> = [:]
+  fileprivate(set) var typeCases: OrderedDictionary<String, TypeCase> = [:]
+  fileprivate(set) var fragments: OrderedDictionary<String, Fragment> = [:]
+
+  init() {}
```

Comment:
is there a reason to have an empty init instead of just using the one with default values?

Reply:
Aha, that makes sense.